sdk/script/management-sdk.ts

```diff
@@ -169,6 +169,11 @@ class AccountManager {
         return this.post(urlEncode `/apps/${appName}/deployments/`, JSON.stringify(deployment), /*expectResponseBody=*/ true)
             .then((res: JsonResponse) => res.body.deployment);
     }
+    
+    public clearDeployment(appName: string, deploymentName: string): Promise<void> {
```

Comment:
Besides a deployment's name, key, and its relationship to its parent app, what other attributes does it have other than its history? I think of the clear operation as being equivalent to clearing an array or dictionary/etc., which wouldn't obviously impact the references that point at that dictionary instance (the app->deployment relationship), or even the "identity" (e.g. memory address of the start of the data structure). The name and key are the identity of the deployment (on the management and acquisition sides, respectively), and I guess I'm just saying that I don't think folks would expect an operation named "clear" to mutate those. But I could obviously be over wrong.

I'm definitely not trying to oversimplify this, but I also don't want to overcomplicate it either. Conceptually thinking of a deployment as a history of releases seems legitimate to me.